tests/run-make/remove-dir-all-race/rmake.rs

```diff
@@ -0,0 +1,52 @@
+//@ ignore-windows
```

Comment:
Suggestions:

1. Could we leave a brief overview of what this test is trying check? Namely, this test is not *guaranteed* to fail even if the underlying impl regresses (that's okay, but we should document it). Even better *why* the check is done this way.
2. We should remark why this is ignore-windows.

Reply:
yeah, that's a good idea, a lot of tests are lacking documentation and are hard to understand because of it.